e2e/LongHaul/service/SystemHealthMessage.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Azure.Devices.LongHaul.Service
 {
-    internal class SystemHealthC2dMessage : MessageBase
```

Comment:
This is not sending to device actually, but more relevant to info of system health. Renamed.